src/i18n/i18n_resources.cljs

```diff
@@ -1,11 +1,7 @@
-(ns status-im.i18n.i18n-resources
+(ns i18n.i18n-resources
```

Comment:
it was already moved `status-im2.setup.i18n-resources`

Reply:
so we need just to remove this, and use new ns in the status-im

Reply:
cool, will adjust 👍